README.md

```diff
@@ -200,7 +200,7 @@ If you want to do development on pyuvdata, in addition to the other dependencies
 you will need the following packages:
 
 * pytest
-* pytest-cases
+* pytest-cases >= 1.12.1
```

Comment:
Should we also add cython to either this list or the main list of dependencies above? (Not sure if the typical user needs cython installed or not...)

Reply:
It is listed as a build dependency in the `pyproject.toml` file. I'm not sure if it is necessary for a user though. I did _try_ to make a blank environment and pip install. Seemed to go okay but maybe someone else has a better test in mind?

Reply:
This should also go in the `environment.yml` file that we point developers to in the readme.